ansible_runner/utils.py

```diff
@@ -186,7 +186,7 @@ class OutputEventFilter(object):
 
     EVENT_DATA_RE = re.compile(r'\x1b\[K((?:[A-Za-z0-9+/=]+\x1b\[\d+D)+)\x1b\[K')
 
-    def __init__(self, handle, event_callback, output_json=False):
+    def __init__(self, handle, event_callback, suppress_ansible_output, output_json=False):
```

Comment:
set `suppress_ansible_output` here to false as a default in case the caller doesn't want to supply this (like `output_json`).

Might also recommend adding a newline before this and letting the latter two parameters sit on the next line